index.src.html

```diff
@@ -61,8 +61,8 @@
 ## Registration (<a>embedded authenticator</a> mode) ## {#registration-embedded}
 
 - On the phone:
-    * User goes to example.com in the browser, and signs in using whatever method they have been using (possibly a legacy
-        method such as a password).
+    * User goes to example.com in the browser and signs in to an existing account using whatever method they have been using
```

Comment:
I know my English is sometimes a bit Russian, but I think we want `into` here
http://blog.oxforddictionaries.com/2015/08/in-to-versus-into/